spec/activerecord-bitemporal/bitemporal_spec.rb

```diff
@@ -1204,6 +1204,21 @@ class EmployeeWithUniquness < Employee
       it { expect { subject }.to change(employee, :transaction_from).from(updated_time).to(destroyed_time) }
       it { expect { subject }.not_to change(employee, :transaction_to) }
     end
+
+    context "with `#force_update`" do
+      subject { Timecop.freeze(destroyed_time) { employee.force_update { employee.destroy } } }
+
+      it { expect { subject }.to change(Employee, :count).by(-1) }
+      it { expect { subject }.to change(employee, :destroyed?).from(false).to(true) }
+      it { expect { subject }.not_to change(employee, :valid_from) }
+      it { expect { subject }.not_to change(employee, :valid_to) }
+      it { expect { subject }.not_to change(employee, :transaction_from) }
+      it { expect { subject }.to change(employee, :transaction_to).from(ActiveRecord::Bitemporal::DEFAULT_TRANSACTION_TO).to(destroyed_time) }
+      it { expect { subject }.not_to change { Employee.ignore_valid_datetime.within_deleted.count } }
```

Comment:
![review:nits](https://img.shields.io/badge/review-nits-green.svg)
It looks like `ignore_bitemporal_datetime` can be used.
```suggestion
      it { expect { subject }.not_to change { Employee.ignore_bitemporal_datetime.count } }
```

Reply:
Makes sense. But `ignore_valid_datetime.within_deleted` is the standard way in the spec.
https://github.com/search?q=repo%3Akufu%2Factiverecord-bitemporal+ignore_valid_datetime.within_deleted&type=code

It would be nice to replace them all in someday.